src/librustc_mir/borrow_check/location.rs

```diff
@@ -1,4 +1,4 @@
-use rustc::mir::{BasicBlock, Location, Mir};
+use rustc::mir::{BasicBlock, Location, Body};
```

Comment:
I don't remember if @oli-obk wanted to leave this for another PR (or maybe I never asked for this before), but IMO `Body` shouldn't be referred to as anything other than `mir::Body` outside of the module it's defined in.